packages/jbrowse-cli/src/base.ts

```diff
@@ -0,0 +1,123 @@
+import Command from '@oclif/command'
+import { promises as fsPromises } from 'fs'
+import * as path from 'path'
+import fetch from 'node-fetch'
+
+export default abstract class JBrowseCommand extends Command {
+  async init() {
+    // const { flags } = this.parse(JBrowseCommand)
+    // this.flags = flags
+  }
```

Comment:
Is this needed?

Reply:
@peterkxie could you weigh in on this

Reply:
oh sorry I missed this, left this in when copying over the oclif custom base class template, probably not needed